filters/shedder/admission.go

```diff
@@ -0,0 +1,366 @@
+package shedder
+
+import (
+	"context"
+	"math"
+	"math/rand"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+	"github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/metrics"
+	"go.uber.org/atomic"
+)
+
+func getIntArg(a interface{}) (int, error) {
+	if i, ok := a.(int); ok {
+		return i, nil
+	}
+
+	if f, ok := a.(float64); ok {
+		return int(f), nil
+	}
+
+	return 0, filters.ErrInvalidFilterParameters
+}
+
+func getDurationArg(a interface{}) (time.Duration, error) {
+	if s, ok := a.(string); ok {
+		return time.ParseDuration(s)
+	}
+
+	i, err := getIntArg(a)
+	return time.Duration(i) * time.Millisecond, err
```

Comment:
I suggest we support duration string only, see https://github.com/zalando/skipper/pull/1814#issuecomment-895053332

Reply:
yeah , just copied it, will change it